dtfs-central-api/src/types/utilisation-reports.ts

```diff
@@ -24,11 +24,18 @@ export type UtilisationReportReconciliationSummary = {
   items: UtilisationReportReconciliationSummaryItem[];
 };
 
-export type ReportPeriodStart = {
+export type ReportPeriodMonthAndYear = {
   month: OneIndexedMonth;
   year: number;
 };
```

Comment:
I think we can now make this a more generic `MonthAndYear` type and move to `date.ts` adjacent to this file.